src/pages/Users/types.tsx

```diff
@@ -0,0 +1,12 @@
+import { UniversalTableProps, BaseData } from "@/components/UniversalTable/types";
+
+export interface User extends BaseData {
+	role: string,
+	fullName: string,
+	password: string
+}
+
+export type UsersPageProps<T> = {
```

Comment:
consider renaming. this name is kinda confusing, because when I read it I expect to see its usage as a prop to a component, not as a type of the loader's data

Reply:
ok, i`ll change naming

Reply:
Done